SpamPkg/Tools/gen_aux/src/auxgen.rs

```diff
@@ -0,0 +1,343 @@
+//! A module that contains the C-equivalent structs and final functions to

```

Comment:
Probably want to call out what/where they are?

Reply:
Added !